docs/preview/features/powershell/azure-api-management.md

```diff
@@ -20,6 +21,34 @@ To have access to the following features, you have to import the module:
 PS> Import-Module -Name Arcus.Scripting.ApiManagement
 ```
 
+## Backing up an API Management service
+
+Backs up an API Management service (with built-in storage context retrieval).
+
+| Parameter             | Mandatory | Description                                                                                                                                                      |
+| --------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`   | yes       | The name of the of resource group under which the API Management deployment exists.                                                                              |
+| `StorageAccountName`  | yes       | The name of the Storage account for which this cmdlet gets keys.                                                                                                 |
+| `ServiceName`         | yes       | The name of the API Management deployment that this cmdlet backs up.                                                                                             |
+| `TargetContainerName` | yes       | The name of the container of the blob for the backup. If the container does not exist, this cmdlet creates it.                                                   |
+| `TargetBlobName`      | no        | The name of the blob for the backup. If the blob does not exist, this cmdlet creates it (default value based on pattern: `{Name}-{yyyy-MM-dd-HH-mm}.apimbackup`. |
+| `PassThru`            | no        | Indicates that this cmdlet returns the backed up PsApiManagement object, if the operation succeeds.                                                              |
```

Comment:
Perhaps use a different, more clear name: _(better suggestions are welcome)_
```suggestion
| `ReturnApimBackupObject`            | no        | Indicates that this cmdlet returns the backed up PsApiManagement object, if the operation succeeds.                                                              |
```

Reply:
I'm wondering why we should need something like this. Could you give me a use case? Thanks